system/lib/libunwind/src/Unwind-wasm.c

```diff
@@ -63,8 +63,14 @@ _LIBUNWIND_EXPORT _Unwind_Reason_Code
 _Unwind_RaiseException(_Unwind_Exception *exception_object) {
   _LIBUNWIND_TRACE_API("_Unwind_RaiseException(exception_object=%p)",
                        (void *)exception_object);
+#if defined(__EMSCRIPTEN__) && !defined(NDEBUG)
```

Comment:
@sbc100 This is currently not working because we set `-DNDEBUG` unconditionally for libunwind: https://github.com/emscripten-core/emscripten/blob/10e5db9fbd992e2449e32116fc3a9e8d2d072041/tools/system_libs.py#L1592

If we make libunwind inherit from [`DebugLibrary`](https://github.com/emscripten-core/emscripten/blob/10e5db9fbd992e2449e32116fc3a9e8d2d072041/tools/system_libs.py#L705-L728) we can make this work, but would it be worth it?

Reply:
Ping 😀

Reply:
I'm not sure its worth it .. WDYT?

Reply:
Don't think so either... Will close it then.